core/dbt/events/types.py

```diff
@@ -33,9 +33,10 @@ def level_tag(self) -> str:
 
 @dataclass
 class ShowException():
-    exc_info: Any = None
-    stack_info: Any = None
-    extra: Any = None
+    def __post_init__(self):
+        self.exc_info: Any = None
+        self.stack_info: Any = None
+        self.extra: Any = None
```

Comment:
Since this runs after the init method, do we have to check that the values haven't been overridden before setting them back to None?

Reply:
We could I suppose, seems a little un-necessary given we always set the values in `cli_msg()` which , by virtue of being a method on the event class will always happen _after_ `__post_init__()`.

Let me know if you still would like that changed, I don't really care about a little extra guard-railing here.

Reply:
oh that makes sense. let's keep it simple then. Maybe add a comment to this effect, but this should be good.

Reply:
👍

Reply:
I added a comment block so it's clear what's happening there.